src/Avalonia.Input/KeyboardDevice.cs

```diff
@@ -23,6 +23,8 @@ public class KeyboardDevice : IKeyboardDevice, INotifyPropertyChanged
         public IInputManager InputManager => AvaloniaLocator.Current.GetService<IInputManager>();
 
         public IFocusManager FocusManager => AvaloniaLocator.Current.GetService<IFocusManager>();
+        
+        public string[] invalidCharacters = new String[1]{""};
```

Comment:
Do we need that `[1]` here? Also, why isn't it a `char` array? Are there any unicode surrogate pairs that we should care of?

Reply:
You can't string.replace with an empty char.